src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
+const SEMICOLON = ';';
+const COLON = ':';
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(SEMICOLON);
+
+  const stylesObject = styles.reduce((temp, style) => {
+    const separateStyle = style.split(COLON).map((part) => part.trim());
+
+    if (separateStyle.length === 2) {
+      temp[separateStyle[0]] = separateStyle[1];
+    }
+
+    return temp;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. In this case, the 'map' method inside the 'reduce' function should be on a new line, indented to align with 'split' in the 'reduce' callback.